pipeline/clean/opuscleaner/configs/en-zh/default.filters.json

```diff
@@ -56,14 +83,6 @@
       },
       "language": null
     },
-    {
```

Comment:
Numbers mismatch was a big problem for our users. It makes translations unreliable and its' hard to catch this issue. I think the right solution here is to fix the Chinese edge case on OpusCleaner side.

Reply:
It does not do a perfect job, but at least it seems that BCAI is able to tell when a numeral is written with letters in the English part and numbers in the Chinese part.
```
September 11, 2001, was a terrible tragedy by any measure, but it was not a historical turning point. It did not herald a new era of international relations in which terrorists with a global agenda prevailed, or in which such spectacular terrorist attacks became commonplace.        2001年9月11日，以任何标准看都是一个可怕的惨剧，但它并非一个历史转折点。它并未开启一个由全球恐怖主义主导国际关系的新时代，类似的大规模恐怖袭击也没有变成家常便饭。虽然人们在“全球反恐战争”上投注了大量精力，但是过去10年最重要的进展是信息技术的创新与传播、全球化、伊拉克与阿富汗战争以及中东的政治动荡。  0.968
NEW YORK – It was a decade ago that 19 terrorists took control of four planes, flew two into the twin towers of the World Trade Center, hit the Pentagon with a third, and crashed the fourth in a field in Pennsylvania after passengers resisted and made it impossible for the terrorists to complete their malevolent mission. In a matter of hours, more than 3,000 innocent people, mostly Americans, but also people from 115 other countries, had their lives suddenly and violently taken from them. 纽约——十年前，19名恐怖分子劫持了四架民航客机，其中两架撞上世贸中心双子塔，第三架撞击了五角大楼，第四架在乘客的反抗下无法完成袭击，坠毁于宾夕法尼亚旷野。只不过几个小时，3000多无辜平民被突如其来的暴行夺去了生命， 其中大多是美国人，但也有来自其他115个国家的人。 0.991
See all 433 hotels in Dominican Republic        查看亨德森维尔的所有 11 家酒店  0.625
Horst Claus Recktenwald Prize in Economics, 2004.       1 2016年诺贝尔经济学奖简介      0.108
Apple iPad Pro 12.9" Retina Display 128GB WiFi Tablet WOB       Apple苹果平板电脑iPad Pro 12.9英寸 深空灰 . . . 0.280
Children will hand over their parents to be put to death. 22 You will be hated by all people because of Me.     10:22 你们还要因我的名,被众人恨恶;惟有忍耐到底的,必然1 得救。   0.180
Our Price: $17.99 Be the first to review this product Reviva Labs Collagen Night Cream -- 1.5 oz (42 g) Our Price: US$16.99 作为评论该产品的第一人 Reviva Labs 祛褐斑美白晚霜 - 1盎司(42克)     0.993
In order to stimulate more investment,the Australian Government now introduce a 5 million investment visa (Significant Investment Visa),And on November 24, 2012 officially opened the application.     为鼓励更多人投资移民澳洲,澳洲政府更推出500万重大投资签证(Significant Investment Visa ) , 并于2012年11月24日正式开放申请。 新出台的500万重大投资签  0.883
Note: By February 2017, all new listings will require at least one of the product identifiers listed above.     ● 到2017年2月,卖家在大部分物品分类中创建新的物品刊登时都将必须提供上面列出的一种以上产品识别码。        0.979
Chemical technology for inorganic substances, materials and goods       化工工艺无机物质,材料和物品 4.5.        0.572
The Corniche Hotel Abu Dhabi is a 30-minute drive from Abu Dhabi International Airport. 酒店内设有免费私人停车场,距离阿布扎比国际机场有25分钟车程。     0.887
iWe closed Broadway from 42nd Street to 47th Street and created two and a half acres of new pedestrian space.   我哋由 42 街至到 47 街封閉 Broadway 劃出新嘅 2.5 英畝嘅行人專用區       0.898
People have ridden seven million miles. 使用里程達到 700 萬英里 0.477
What I and my colleagues did was put 32 people, who were madly in love, into a functional MRI brain scanner.    放入一部大腦功能磁力共振掃描儀  0.060
```

Reply:
Ok, we can train a model and test this